### PR TITLE
Updates the README file to include outExtension.

### PR DESCRIPTION
**Pull request by muellercodes**

### CODE REVIEW

**Comment by muellercodes** [comment]
Hmm I haven't made any changes to the repo code itself, just the README, yet this fails the tests.

**Comment by cheshire137** [comment]
I wonder if my Semaphore CI setup is broken; I'll ignore the red build. Thank you for the PR!

File: README.md [hunk 1]
```diff
@@ -99,6 +103,18 @@ Use the `encodings` option to specify encodings, e.g., `{encodings: "UTF-8"}`.
            pipe(gulp.dest('./public'));
     });
 
+    // Change the output extension
+    gulp.task("haml-to-handlbars", function() {
```
Comment (cheshire137) [inline README.md]: Minor typo: `handlbars` => `handlebars`

**Review by cheshire137 — APPROVED** [review]
Thanks!

**Pull request merged**